### ISSUE TITLE
Support for nette/utils:^4.0

### ISSUE THREAD

**Issue by lupo112** [nextras/orm#608]
Current version is not supporting latest versions of nette.

Problematic package: `nette/utils:^4.0`

```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires nextras/orm ^4.0 -> satisfiable by nextras/orm[v4.0.0, ..., v4.0.6].
    - nextras/orm[v4.0.0, ..., v4.0.6] require nette/utils ~2.5 || ~3.0 -> found nette/utils[v2.5.0, ..., v2.5.7, v3.0.0, ..., v3.2.9] but it conflicts with your root composer.json require (^4.0).
```

Can you add support for this version?

Used `composer.json`:
https://github.com/nette/web-project/blob/v3.1.13/composer.json

**Comment by hrach** [comment]
@lupo112 Are you interested in support in main or for stable version?

**Comment by hrach** [comment]
I fear there can me more issue 😨 as the whole stack would be required to be updated.

**Comment by lupo112** [comment]
the problem is only on `nette/utils` package, cannot it be used like this?

```
"nette/utils": "~3.0 || ~4.0",
```

is there any conflicts with the new nette/utils 4.0 version?

**Comment by radimvaculik** [comment]
@hrach Fixed @ d364f8066e08d91cfe00ec0e55003ff94cd3c862, right?

**Comment by hrach** [comment]
This issue is for 4.0 release. PR welcomed, imo nothing difficult.

**Comment by mskocik** [comment]
@hrach any ETA on releasing new version with this support?

**Comment by harmim** [comment]
Can anyone please create a new release of the package that would include this change so one would not depend on the main version of the package?

**Comment by hrach** [comment]
released as 4.0.7
